### PR TITLE
add support to replica mode for redis-sentinel to find the master

### PR DESCRIPTION
**Pull request by ermeaney**
We are using redis with redis-sentinel in mesos and marathon, and being able to scale or restart redis containers and have them get their master from sentinel seems like a good pattern

this PR adds support for redis-sentinel to the 5.0 redis 

I have updated the README with an example of usage

### CODE REVIEW

**Comment by javsalgar** [comment]
Thanks for your PR. We will review it in these days :)

File: README.md [hunk 1]
```diff
@@ -377,6 +377,21 @@ The above command scales up the number of replicas to `3`. You can scale down in
 
 > **Note**: You should not scale up/down the number of master nodes. Always have only one master node running.
 
+### Using redis-sentinel
+
+If you have redis-sentinel running, you can use it to find the current master
```
Comment (javsalgar) [inline README.md]: I would add a bit more saying which env vars allow this

Comment (ermeaney) [reply README.md]: sure, just updated the PR with some more details

**Comment by ermeaney** [comment]
I add the requested changes

**Comment by javsalgar** [comment]
Thank you very much. We will integrate and test the changes :)

**Pull request merged**